latest_changes/main.py

```
@@ -233,9 +233,9 @@ def main() -> None:
         sys.exit(1)
 
     logging.info("Setting up GitHub Actions git user")
-    subprocess.run(["git", "config", "user.name", "github-actions"], check=True)
+    subprocess.run(["git", "config", "user.name", "github-actions[bot]"], check=True)
```

Ruff (0.15.7)

[error] Starting a process with a partial executable path

(S607)